influx/annotations.go

```diff
@@ -0,0 +1,265 @@
+package influx
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"sort"
+	"time"
+
+	"github.com/influxdata/chronograf/id"
+
+	"github.com/influxdata/chronograf"
+)
+
+const (
+	// AllAnnotations returns all annotations from the chronograf database
+	AllAnnotations = `SELECT "start_time", "modified_time_ns", "text", "type", "id" FROM "chronograf"."autogen"."annotations" WHERE "deleted"=false AND time >= %dns and "start_time" <= %d ORDER BY time DESC`
+	// GetAnnotationID returns all annotations from the chronograf database where id is %s
+	GetAnnotationID = `SELECT "start_time", "modified_time_ns", "text", "type", "id" FROM "chronograf"."autogen"."annotations" WHERE "id"='%s' AND "deleted"=false ORDER BY time DESC`
```

Comment:
dbrp and measurement probably shouldn't be used explicitly here.

Reply:
Fixed in a8b52523e9a972f7cb5aeb463b6a178cef1edafa